manifests1/go.sum

```diff
@@ -0,0 +1,98 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
why are you pushing go.som files ?

Reply:
*sum

Reply:
removing

Reply:
@sawood14012 I run ui tests after deleting go.sum, all CA and SA test failed for Go ecosystem.

Reply:
no, sorry something is wrong in go ecosystem in prod I guess

Reply:
> > i guess whenever it runs it makes use of Production env please change it to staging these automated tests should work on stage @immanishbainsla
> 
> looking into it.
> but a question in general, as we are doing UI tests (on behalf of user) so prod should be the preference as user interacts with prod not staging.

nope no test should run on prod expect hourly test